baseframe/templates/baseframe/bootstrap3/autoform.html.jinja2

```diff
@@ -14,10 +14,10 @@
     $(function() {
       {{ widgetscripts(form, script=false) }}
       // Reset disabled state if page was reloaded (TODO: does this really get called?)
-      $('#{{ formid }} .submit-disabled').prop('disabled', false);
+      $('#{{ ref_id }} .submit-disabled').prop('disabled', false);
```

Comment:
`ref_id` will change here if this is after the reload. Test if this line ever gets called at all.

Reply:
Removed this line since it isn't being called